### PR TITLE
Merge new modifications in main

### PR DESCRIPTION
**Pull request by lorenz10**

### CODE REVIEW

**Comment by lorenz10** [comment]
Sorry wrong repo....

File: lib/screens/about.dart [hunk 1]
```diff
@@ -148,6 +148,17 @@ class _AboutScreenState extends State<AboutScreen> {
...
                             ),
+                            Divider(),
+                            SizedBox(height: 10),
+                            Text(
+                              'App illustrations thanks to the courtesy of DesignAi',
+                            ),
+                            TextButton(
+                              onPressed: () async => launchMailto(),
+                              child: Text(
+                                'Link da inserire...',
```
Comment (bananenwilly) [inline lib/screens/about.dart:159]: Surprise Italian?

Comment (lorenz10) [reply lib/screens/about.dart:159]: I fixed in the newest merge, this was a mistake...

Comment (lorenz10) [reply lib/screens/about.dart:159]: "Italian surprise" ahahah

**Pull request closed without merging**